src/openforms/js/components/admin/form_versions/FormVersionsTable.js

```diff
@@ -0,0 +1,78 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import useAsync from 'react-use/esm/useAsync';
+
+import {get, apiCall} from '../../../utils/fetch';
+import {FORM_ENDPOINT} from '../form_design/constants';
+import Loader from "../Loader";
+
+
+const FormVersionsTable = ({ csrftoken, formUuid, formAdminUrl}) => {
+    const [formVersions, setFormVersions] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    const restoreVersion = async (csrftoken, formUuid, versionUuid, redirectUrl) => {
+        await apiCall(
+            `${FORM_ENDPOINT}/${formUuid}/versions/${versionUuid}/restore`,
+            {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                    'X-CSRFToken': csrftoken,
+                },
+            }
+        );
```

Comment:
Is there a reason you're not using the `post` function from `utils/fetch`?

Reply:
I didn't want the second part of the fetch, since no body is returned: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/utils/fetch.js#L43

Reply:
You can adapt the `fetch` call to only get the JSON if there is a response body and otherwise return `null`. That's a good fix/cleanup.